src/output/icons.rs

```diff
@@ -113,12 +115,19 @@ impl Icons {
 /// Mapping from full filenames to directory icon. This mapping should contain
 /// all the directories that have a custom icon.
 const DIRECTORY_ICONS: Map<&'static str, char> = phf_map! {
+    "__pycache__"         => Icons::LANG_PYTHON,    // 󰌠
+    ".cargo"              => Icons::LANG_RUST,      // 
     ".config"             => Icons::FOLDER_CONFIG,  // 
+    ".doom.d"             => Icons::EMACS,          // 
+    ".emacs.d"            => Icons::EMACS,          // 
     ".git"                => Icons::FOLDER_GIT,     // 
     ".github"             => Icons::FOLDER_GITHUB,  // 
+    ".gnupg"              => Icons::FOLDER_KEY,     // 󰢬
+    ".local"              => Icons::FOLDER_CONFIG,  // 
     ".npm"                => Icons::FOLDER_NPM,     // 
     ".ssh"                => Icons::FOLDER_KEY,     // 󰢬
     ".Trash"              => '\u{f1f8}',            // 
+    "bin"                 => Icons::FOLDER_CONFIG,  // 
```

Comment:
bin is not a configuration folder

Reply:
Sorry it seems like a common folder that people would have